Dockerfile

```diff
@@ -140,6 +140,9 @@ RUN source /etc/default/evm && \
     yarn install && \
     gulp build
 
+## Clean the logs
+RUN rm -rf ${APP_ROOT}/log/*
```

Comment:
Do we have this problem on appliance image builds?  If so, and if we are already handling it, do we use the technique that Geert used to use a tmpfs disk, so that it keeps the VM image size smaller?